src/client/app/util/download.ts

```diff
@@ -17,67 +19,49 @@ export const downloadUrls = async (urlCount: number, tableConfig: any) => {
     'Created At',
     'Last Modified\n',
   ])
-
-  const limit = 100
-  // gets 100 urls at a time
-  const numOfGetRequests = Math.ceil(urlCount / limit)
-
-  const getUrlsPromiseArray = []
-
-  // eslint-disable-next-line no-plusplus
-  for (let i = 0; i < numOfGetRequests; i++) {
-    const queryObj = {
-      ...tableConfig,
-      limit,
-      offset: i * limit,
-    }
-    getUrlsPromiseArray.push(userActions.getUrls(queryObj))
-  }
-
-  await Promise.all(getUrlsPromiseArray).then((promises) => {
+  const queryObj = queryObjFromTableConfig(tableConfig)
+  await userActions.getUrls(queryObj).then((promise) => {
```

Comment:
nit: would it be preferable to do the following? it looks possible to eliminate the `then` callback entirely by taking the result of `await`

```suggestion
  const { json, isOk } = await userActions.getUrls(queryObj)
```

Reply:
good idea, done!